include/vkgcDefs.h

```diff
@@ -687,6 +688,24 @@ struct SamplerYCbCrConversionMetaData {
     };
     unsigned u32All;
   } word3;
+
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION > 50
```

Comment:
We don't need to check major version since you just bump minor version and assume the newly-added codes have no impacts on existing interface.